src/pilot/deploy-overcloud.py

```diff
@@ -464,6 +472,14 @@ def main():
         if args.dvr_enable:
             env_opts += " -e ~/pilot/templates/neutron-ovs-dvr.yaml"
 
+        # The octavia.yaml must be included after the
+        # network-environment.yaml
+        if args.octavia_enable:
+            env_opts += " -e ~/pilot/templates/octavia.yaml"          
+
+        if args.octavia_user_certs_keys is True:
+            env_opts += " -e ~/pilot/templates/cert_keys.yaml"
+
```

Comment:
You probably only want to enable the cert yaml if octavia is enabled as well.

Reply:
Yes you are right. This is fixed now.